_test_/index.test.tsx

```diff
@@ -1,9 +1,20 @@
-﻿import React from 'react';
+﻿import React, {FunctionComponent} from 'react';
 import {render} from '@testing-library/react';
 import Home from "../pages";
+import { RouterContext } from 'next-server/dist/lib/router-context'
 
-test('Check that the home page renders', () => {
-    const home = render(<Home/>);
+
+test('Check that the home page renders', async () => {
+   const router = {
+       query:{token: "undefined"},
+   };
+    const home = render(
+        <RouterContext.Provider value={router.query}>
+            <Home/>
+        </RouterContext.Provider>
+        )
+    ;
```

Comment:
Nice - looks great.
Could you change the token to an actual value though rather than `undefined`?